### PR TITLE
template per module

### PR DESCRIPTION
**Pull request by lemingos**
#144 Fix

### CODE REVIEW

File: marketplace-kit-archive.js [hunk 1]
```diff
@@ -90,9 +95,8 @@ const makeArchive = (path, directory, withoutAssets) => {
   archive.glob('**/*', options, { prefix: directory });
 
   addModulesToArchive(archive).then(r => {
-    setTimeout(() => {
-      archive.finalize();
-    }, 500);
+    console.log("finalize");
```
Comment (ignacy) [inline marketplace-kit-archive.js]: Use logger

File: marketplace-kit-archive.js [hunk 2]
```diff
@@ -31,17 +31,22 @@ const addModuleToArchive = (module, archive) => {
       if (err) throw reject(err);
       const moduleTemplateData = templateData(`modules/${module}/template-values.json`);
 
-      for (f of files) {
-        const path = `modules/${module}/${f}`;
-        fs.lstat(path, (err, stat) => {
-          if (!stat.isDirectory()) {
-            archive.append(templates.fillInTemplateValues(path, moduleTemplateData), {
-              name: path
+      return Promise.all(
```
Comment (ignacy) [inline marketplace-kit-archive.js]: TIL :wink:

File: marketplace-kit-archive.js [hunk 3]
```diff
@@ -31,17 +31,22 @@ const addModuleToArchive = (module, archive) => {
...
+        files.map(f => {
+          const path = `modules/${module}/${f}`;
+          return new Promise((resolve, reject2) => {
```
Comment (ignacy) [inline marketplace-kit-archive.js]: `reject2`? Where is `reject1` ?

Comment (lemingos) [reply marketplace-kit-archive.js]: ✂️

File: marketplace-kit-archive.js (hunk 1 above)
Comment (lemingos) [reply marketplace-kit-archive.js]: This is no longer there

**Pull request merged**